frontend/src/components/layout/whatsnew/WhatsNewMenu.tsx

```diff
@@ -160,6 +162,31 @@ export const WhatsNewMenu = (props: Props) => {
     entries.push(signBackInEntry);
   }
 
+  const aliasToMask: WhatsNewEntry = {
+    title: l10n.getString("whatsnew-feature-alias-to-mask-heading"),
+    snippet: l10n.getString("whatsnew-feature-alias-to-mask-snippet"),
+    content: (
+      <WhatsNewContent
+        description={l10n.getString(
+          "whatsnew-feature-alias-to-mask-description"
+        )}
+        heading={l10n.getString("whatsnew-feature-alias-to-mask-heading")}
+        image={aliasToMaskHero.src}
+      />
+    ),
+    hero: aliasToMaskHero.src,
+    icon: aliasToMaskIcon.src,
+    dismissal: useLocalDismissal(
+      `whatsnew-feature_sign-back-in_${props.profile.id}`
+    ),
```

Comment:
I'm unsure as to what this `dismissal` object is for, and why all of the news items have it set to `whatsnew-feature_sign-back-in_${.....`

Reply:
[It is](https://github.com/mozilla/fx-private-relay/blob/ab13f0ccec87ec56bbf6e6e561a376afbf30d728/frontend/src/hooks/localDismissal.ts#L22) the ID that will be used for the cookie that tracks whether the current user has already seen this news entry. So you'll want to use an ID here that is unique for this entry, e.g.

```suggestion
    dismissal: useLocalDismissal(
      `whatsnew-feature_alias-to-mask_${props.profile.id}`
    ),
```